Equation.java

```diff
@@ -0,0 +1,49 @@
+import static java.lang.Math.*;
+/**
+ * solve the quadratic equation
+ * @author Babak Alexey
+ */
+public class Equation {
+
+   /** 
+    * check possibility of the "String -> Double" conversion
+    * @param string  the string to check,
+    * if string has only double variable  - return true
+    */
+   public static boolean checkString(String string) {
+      try {
+         Double.parseDouble(string);
+      } catch (Exception e) {
+           return false;
+      }
+      return true;
+   }
+
+   /** 
+    * solve the quadratic equation 
+    */
+   public static void main(String [] args) {
+      if (args.length != 3) {
+         System.out.println("Try to input 3 numbers ( a,b,c)");
+         System.exit(0);
+      } else if (!checkString(args[0]) || !checkString(args[1])|| !checkString(args[2])) {
+         System.out.println("Try to input 3 numbers, not letters or other symbols");
+         System.exit(0);
+      }
+      double a = Double.parseDouble(args[0]);

```

Comment:
You should perform parseDouble only one time for each argument and check for exception